Authentication/Extensions.swift

```diff
@@ -2,20 +2,42 @@
 //  Extensions.swift
 //  Authentication
 //
-//  Created by Daniela Riesgo on 3/7/16.
+//  Created by Daniela Riesgo on 7/25/16.
 //  Copyright © 2016 Wolox. All rights reserved.
 //
 
 import Foundation
 
 public extension String {
     
-    var localized: String {
-        return NSLocalizedString(self, tableName: nil, bundle: NSBundle.mainBundle(), value: "", comment: "")
+    public var frameworkBundle: NSBundle { return  NSBundle(forClass: LoginView.self) }
```

Comment:
Also, I don't think frameworkBundle should be a var in an extension of String. It should be a free let.